.github/workflows/tests.yml

```diff
@@ -29,11 +31,13 @@ jobs:
           - {name: "debian", tag: "testing"}
           - {name: "debian", tag: "12"}
           - {name: "fedora", tag: "rawhide", url: "registry.fedoraproject.org/"}
+          - {name: "fedora", tag: "42", url: "registry.fedoraproject.org/"}
           - {name: "fedora", tag: "41", url: "registry.fedoraproject.org/"}
           - {name: "fedora", tag: "40", url: "registry.fedoraproject.org/"}
```

Comment:
The description mentions dropping some distros, but I don't see anything being removed here. Shouldn't this line be removed?

Reply:
Oops, forgot to push after the last commit. Line removed. Thanks!